controllers/host/host_controller.go

```diff
@@ -2470,21 +2473,23 @@ func (r *HostReconciler) GetAddressPoolsFromPlatformNetwork(associated_addrpools
 	return addrpool_instances, errs
 }
 
-// LockHostRequest takes the host object, host ID / personality and reconcile_object
-// which describes the purpose of requesting locking of particular host, starts lock
-// host monitor and returns LockedDisabledHost monitor error.
+// LockHostRequest takes the host object, host ID / personality and
+// starts lock host monitor and returns LockedDisabledHost monitor error.
 // This would initiate lock request of a host through VIM so that disabled attributes
 // can be reconciled later.
-func (r *HostReconciler) LockHostRequest(host_instance *starlingxv1.Host, host_id string, host_personality string, reconcile_object string) error {
+func (r *HostReconciler) LockHostRequest(host_instance *starlingxv1.Host, host_id string, host_personality string, set_res_info bool) error {
```

Comment:
I only see the SetWaitForStrategySent(true) in this method, but the LockHostReqeust() doesn't cover all the reconfiguration path

Reply:
In case of oam network reconfiguration the procedure is:
1. Reconfigure oam
2. Lock and unlock controllers
So, the reconfiguration is already done prior to lock / unlock in this case.

Reply:
Can we have it as "LockHostRequestByOtherController/reconciler"? We are not using it in the host reconciler, right?
Can we have an info message to indicate the sent and update operation? note: we don't want it repeatedly dump into the logs, expected to have some prechecks

Reply:
Done